src/config/wmodules-ciscat.c

```diff
@@ -41,6 +42,13 @@ int wm_ciscat_read(const OS_XML *xml, xml_node **nodes, wmodule *module)
     ciscat->flags.scan_on_start = 1;
     sched_scan_init(&(ciscat->scan_config));
     ciscat->scan_config.interval = WM_DEF_INTERVAL;
+    
+    // Set default ciscat binary
+    #ifdef WIN32
+        ciscat->ciscat_binary = WM_CISCAT_V3_BINARY_WIN;
+    #else
+        ciscat->ciscat_binary = WM_CISCAT_V3_BINARY;
+    #endif
```

Comment:
According to the change below, store these strings in the heap.
```suggestion
    #ifdef WIN32
        os_strdup(WM_CISCAT_V3_BINARY_WIN, ciscat->ciscat_binary);
    #else
        os_strdup(WM_CISCAT_V3_BINARY, ciscat->ciscat_binary);
    #endif
```